language/en/INGAME.php

```diff
@@ -83,7 +83,7 @@
 $LNG['lm_officiers']						= 'Officers';
 $LNG['lm_trader']							= 'Market';
 $LNG['lm_fleettrader']						= 'Merchant of Fleets';
-$LNG['lm_technology']						= 'Technologies';
+$LNG['lm_technology']						= 'Technology';
```

Comment:
Every translation has plural of this word. Most of submenus are plural too.

Reply:
Interesting ... any idea why that is? In normal spoken English, "technologies" isn't at all used. It's still a valid word, though generally the plural in English IS "Technology". "We're researching new technology" is a regular sentence, but also "We're researching many technologies" - and neither one is specifically plural - because we in the States are absolutely insane. I'll cut that one though back to what it was for the PR, because it's definitely not an issue either way.